app/admin/contributors.rb

```diff
@@ -0,0 +1,11 @@
+ActiveAdmin.register Contributor do
```

Comment:
Why do we need this new admin page?

As the admin I can't think how a list of contributors is useful. I only need to see it in conjunction with their contribution.